.github/workflows/graalvm.yml

```diff
@@ -17,39 +17,59 @@ jobs:
   build:
     if: github.repository != 'micronaut-projects/micronaut-project-template'
     runs-on: ubuntu-latest
+    env:
+      OCI_CLI_USER: ${{ secrets.OCI_USER }}
+      OCI_CLI_TENANCY: ${{ secrets.OCI_TENANCY }}
+      OCI_CLI_FINGERPRINT: ${{ secrets.OCI_FINGERPRINT }}
+      OCI_CLI_KEY_CONTENT: ${{ secrets.OCI_KEY_FILE }}
+      OCI_CLI_REGION: ${{ secrets.OCI_REGION }}
     strategy:
       matrix:
         java: [ '17' ]
         graalvm: ['latest']
-        k8s: [ '1.21' ]
     steps:
       - uses: actions/checkout@v3
+      - name: Configure Kubectl
+        uses: oracle-actions/configure-kubectl-oke@v1.3.1
+        id: test-configure-kubectl-oke-action
+        with:
+          cluster: ${{ secrets.OKE_CLUSTER_OCID }}
+      - name: Run Kubectl
+        run: kubectl get nodes -A
       - name: Pre-build
-        uses: ./.github/actions/gradle-pre-build
+        uses: ./.github/actions/graalvm-pre-build
         id: pre-build
         env:
-          K8S_VERSION: ${{ matrix.k8s }}
           GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
           GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GRADLE_ENTERPRISE_CACHE_USERNAME }}
           GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GRADLE_ENTERPRISE_CACHE_PASSWORD }}
+          GIT_COMMIT_HASH: ${{ github.sha }}
+          OCI_REGION: ${{ secrets.OCI_REGION }}
+          JOB_ID: ${{ format('{0}-graalvm-java-{1}', github.run_id, matrix.java) }}
+          OCI_TENANCY_NAME: ${{ secrets.OCI_TENANCY_NAME }}
+          JAVA_VERSION: ${{ matrix.java }}
+          AUTH_TOKEN: ${{ secrets.OCI_AUTH_TOKEN }}
+          OCIR_USERNAME: ${{ secrets.OCIR_USERNAME }}
```

Comment:
Can we change the action to use `OCI_CLI_USER ` instead?

Also, what is the difference between `secrets.OCIR_USERNAME ` and `secrets.OCI_USER `

Reply:
The `OCI_USER` contains the OCID of user.
The `OCIR_USERNAME`  contains the username of user.